ops/testing.py

```diff
@@ -1495,7 +1495,14 @@ def push(
 
     def list_files(self, path: str, *, pattern: str = None,
                    itself: bool = False) -> typing.List[pebble.FileInfo]:
-        files = [self._fs.get_path(path)]
+        try:
+            files = [self._fs.get_path(path)]
+        except FileNotFoundError as e:
```

Comment:
The linter is upset about the "as e". Once you get rid of that, we should be good to go!

Reply:
huh? this passed the last... N times. Sure.